packages/components/pages/f-loyalty/package.json

```diff
@@ -73,9 +73,6 @@
     "@justeat/f-trak": "0.x",
     "@justeat/f-wdio-utils": "1.x",
     "@vue/cli-plugin-babel": "4.5.16",
-    "@vue/cli-plugin-unit-jest": "4.5.16",
-    "@vue/test-utils": "1.0.3",
-    "faker": "5.5.3",
-    "miragejs": "0.1.42"
+    "faker": "5.5.3"
```

Comment:
I think in some other packages you pulled this one out too?

Reply:
Ah yeah. I didn't remove this yet as it'll be updating all the imports in `f-loyalty` to `@fakerjs/faker`. I didn't want to add anymore file changes.

I've tested locally though with the new namespaced dep and it work fine, so happy to PR this once the is merged!